wire-ios/Wire-iOS/Sources/UserInterface/Settings/Debug Report/SettingsDebugReportViewController.swift

```diff
@@ -0,0 +1,168 @@
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = SemanticColors.View.backgroundDefault
+
+        setupViews()
+        setupConstraints()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        setupNavigationBarTitle()
+    }
+
+    // MARK: - Setup
+
+    private func setupViews() {
+        view.addSubview(infoLabel)
+        view.addSubview(sendReportButton)
+        view.addSubview(shareReportButton)
+    }
+
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            infoLabel.topAnchor.constraint(equalTo: view.safeTopAnchor, constant: LayoutConstants.padding),
+            infoLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: LayoutConstants.padding),
+            infoLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -LayoutConstants.padding),
+
+            shareReportButton.bottomAnchor.constraint(equalTo: view.safeBottomAnchor, constant: -LayoutConstants.safeBottomPadding),
+            shareReportButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: LayoutConstants.padding),
+            shareReportButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -LayoutConstants.padding),
+            shareReportButton.heightAnchor.constraint(greaterThanOrEqualToConstant: LayoutConstants.buttonHeight),
+
+            sendReportButton.bottomAnchor.constraint(equalTo: shareReportButton.topAnchor, constant: -LayoutConstants.spacing),
+            sendReportButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: LayoutConstants.padding),
+            sendReportButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -LayoutConstants.padding),
+            sendReportButton.heightAnchor.constraint(greaterThanOrEqualToConstant: LayoutConstants.buttonHeight)
+        ])
+    }
+
+    private func setupNavigationBarTitle() {
+        // Set the navigation item title
+        navigationItem.title = Strings.TechnicalReportSection.title.capitalized
+
+        // Set the navigation bar title color and font
+        let titleTextAttributes: [NSAttributedString.Key: Any] = [
+            .foregroundColor: SemanticColors.Label.textDefault,
+            .font: UIFont.font(for: .h3)
+        ]
+        navigationController?.navigationBar.titleTextAttributes = titleTextAttributes
+
+        // Ensure title supports LargeContentViewer
+        navigationItem.titleView?.isAccessibilityElement = true
+        navigationItem.titleView?.accessibilityTraits = .header
+        navigationItem.titleView?.accessibilityLabel = navigationItem.title
+
+        if let titleView = navigationItem.titleView {
+            titleView.showsLargeContentViewer = true
+            titleView.largeContentTitle = navigationItem.title
+        }
+    }
```

Comment:
a new method has beed introduced inside WireReusableComponents: We can use the `setupNavigationBarTitle(Strings.TechnicalReportSection.title.capitalized)` and we get all of this functionality.

Reply:
https://github.com/wireapp/wire-ios/pull/1658/commits/5314a9766a52b49606900f6b32a0fd242067a78e